.storybook/manager.js

```diff
@@ -1,6 +1,6 @@
-import {addons} from '@storybook/addons';
+import { addons } from '@storybook/manager-api';
```

Comment:
Version update migration

Reply:
Would it make sense to lock the Storybook version in package.json to avoid accidental major updates causing breaking changes?

Reply:
It is all checked in CI, so I feel good with it. This was deprecated for many versions.